HigherLevelInterface:
  update_rate: 6                             
  publisherTopic : "/lifterFeedback"                         
  subscriberTopic : "/lift_action"                         
  numberOfReadRegisters : 10
  triggerFailureThreshold : 20
  tasksFileName : " "                   
